test/metabase/models/action_test.clj

```diff
@@ -75,3 +73,21 @@
             (is (= 1 (t2/count DashboardCard :id dashcard-id)))
             (t2/delete! Action :id action-id)
             (is (zero? (t2/count DashboardCard :id dashcard-id)))))))))
+
+(deftest create-update-select-implicit-action-test
```

Comment:
I created this test because it covers the new logic in `action/insert!` and `action/update!` where the `database_id` is stripped out before persisting the data.